src/Calculator/Views/GraphingCalculator/GraphingCalculator.xaml

```diff
@@ -30,6 +52,30 @@
         <Grid x:Name="LeftGrid"
               Grid.Row="1"
               Grid.Column="0">
+            <Grid.Resources>
```

Comment:
Unfortunately, XAML doesn't provide a elegant way to include resources in Style, so we need to be hacky :(

But I would recommend to limit the scope of these brushes a little more to be sure to not affect Buttons and RepeatButtons in the Grapher control and all future controls in this Grid.

Two proposals:
- Limit the scope to the buttons:
    - create a ResourceDictionary file with these brushes 
    - write: `<RepeatButton><RepeatButton.Resources><ResourceDictionary Source="ZoomButtonResourceDictionary.xaml" /></RepeatButton.Resources>`
- Create a new Grid hosting the buttons and move these resources there